wdl/src/test/scala/wdl/types/WdlArrayTypeSpec.scala

```diff
@@ -0,0 +1,34 @@
+package wdl.types
+
+import org.scalatest.{FlatSpec, Matchers}
+import wdl.WdlExpression
+import wdl.expression.NoFunctions
+import wom.types._
+import wom.values.{WdlArray, WdlValue}
+
+import scala.util.{Failure, Success}
+
+class WdlArrayTypeSpec extends FlatSpec with Matchers  {
```

Comment:
This and the following were broken off the WOM tests since they do very WDL specific things.